tests/test_lint_recipe.py

```diff
@@ -1312,7 +1306,7 @@ def test_version(self):
             "Package version 2.0.0~alpha0 doesn't match conda spec"
         )
         lints, hints = linter.lintify_meta_yaml(meta)
-        self.assertIn(expected_message, lints)
+        assert any(expected_message in lint for lint in lints)
```

Comment:
If we replace this, it should be
```suggestion
        assert expected_message in lints
```

However, I see a lot of `self.assertIn` that aren't changed here, so I'm wondering what's the rationale for this change?

(if the above suggestion fails for whatever reason, match the expected message exactly)

Reply:
The lint now includes error details that are generated by our dependency `conda` (see the other comment in which I tagged you). Therefore, your suggestion fails. And also, I did not want to match the expected message exactly because `conda` could change their error message slightly and then our tests fails.